test/bigtest/tests/job-profile-test.js

```diff
@@ -13,15 +13,18 @@ import {
   jobProfiles,
 } from '../interactors';
 
-async function setupFormSubmitErrorScenario(server, responseData = {}) {
+async function setupFormSubmitErrorScenario(method, server, responseData = {}) {
   const {
     response = {},
     status = 500,
     headers = {},
   } = responseData;
 
-  server.put('/data-import-profiles/jobProfiles/:id', () => new Response(status, headers, response));
-  await jobProfileForm.nameFiled.fillAndBlur('Changed');
+  const url = `/data-import-profiles/jobProfiles${method === 'put' ? '/:id' : ''}`;
```

Comment:
This is how we prefer to format ternary operator:
```js
method === 'put'
  ? '/:id'
  : ''
```

Reply:
If a ternary operator does not take a lot of space it is okay to write it in one line. See point 9 in rules https://wiki.folio.org/display/FOLIJET/Code+style+rules+not+covered+by+eslint